editor/debugger/debug_adapter/debug_adapter_protocol.cpp

```diff
@@ -32,6 +32,7 @@
 
 #include "core/config/project_settings.h"
 #include "core/debugger/debugger_marshalls.h"
+#include "core/debugger/remote_debugger.h"
```

Comment:
```suggestion
```
Unnecessary, already included via the header

Reply:
Fixed :+1: